composer.json

```diff
@@ -26,6 +26,7 @@
     "google/auth": "0.7",
     "guzzlehttp/guzzle": "~5.2|~6.0",
     "guzzlehttp/psr7": "1.2.*",
+    "monolog/monolog": "~1",
```

Comment:
I'm not super concerned re: hard dependency here. Maybe it's more elegant to use a "suggest" section since it's in fact no hard dependency to the library, just for a specific feature (as of now).